examples/siro_sandbox/sandbox_app.py

```diff
@@ -681,15 +740,63 @@ def get_grasp_release_controls_text():
             text_delta_y=-50,
         )
 
-    def sim_update(self, dt):
-        # todo: pipe end_play somewhere
+    def record_task_state(self):
+        agent_states = []
+        for agent_idx in range(self.ctrl_helper.n_robots):
+            art_obj = (
+                self.get_sim().agents_mgr[agent_idx].articulated_agent.sim_obj
+            )
+            rotation_quat = mn.Quaternion.from_matrix(
+                art_obj.transformation.rotation()
+            )
+            rotation_list = list(rotation_quat.vector) + [rotation_quat.scalar]
+            pos = art_obj.transformation.translation
+
+            snap_idx = self.env.task._sim.agents_mgr._all_agent_data[
+                agent_idx
+            ].grasp_mgr.snap_idx
+
+            agent_states.append(
+                {
+                    "position": pos,
+                    "rotation_xyzw": rotation_list,
+                    "grasp_mgr_snap_idx": snap_idx,
+                }
+            )
+
+        self._step_recorder.record("agent_states", agent_states)
+
+        self._step_recorder.record(
+            "target_object_positions", self.get_target_object_positions()
+        )
+
+    def record_action(self, action):
+        action_args = action["action_args"]
+
+        keys_to_clear = [
+            "human_joints_trans",
```

Comment:
This is a pretty big array of numbers for every step and I saw it massively bloating our file sizes. I also don't think it will be useful data for recomputing metrics. Meanwhile, for visualization, we should expect to use gfx-replay (separate from this PR).

Reply:
I agree this is unnecessary.

Reply:
Let's then exclude all the keys where `human_joints_trans` in action_arg_key.